docker/snap-breakout/breakoutSnap.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+import argparse
+import h5py
+import pandas as pd
+
+def main():
+    parser = argparse.ArgumentParser(description="Extract the data in a snap file as csv files")
+    parser.add_argument('--input-file', dest='inputfile', help='snap input file')
+    parser.add_argument('--output-prefix',dest='output_prefix', help='output file prefix',default='')
+    args = parser.parse_args()
+
+    #inputfile = 'atac_v1_adult_brain_fresh_5k.snap'
+    #hd5file = h5py.File('output-big.snap')
+
+    hd5file = h5py.File(args.inputfile)
+
+    # HD = hd5file['HD']
+    # if not HD['MG'][()] == 'SNAP':
+    #    raise('The input is not a SNAP file')
+
+    # ###########################
```

Comment:
```suggestion
    ############################
```

Making it consistent with other headings.